daemon/container.go

```diff
@@ -1034,28 +1034,23 @@ func (container *Container) Copy(resource string) (io.ReadCloser, error) {
 		}
 	}()
 
+	if len(resource) == 0 { // Copying `/` we expand to copy its content
```

Comment:
Not sure, but are you sure we wouldn't be better off generating an error? Its like a "cp" w/o a src path - kind of odd. And some people may think that "" should mean "current dir" and not "/".